src/Simplic.Flow.Node/Simplic.Flow.Node.csproj

```diff
@@ -1,78 +1,83 @@
+    <Optimize>false</Optimize>
+    <OutputPath>bin\Debug\</OutputPath>
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\Release\</OutputPath>
+    <DefineConstants>TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Newtonsoft.Json, Version=11.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+      <HintPath>..\packages\Newtonsoft.Json.11.0.2\lib\net45\Newtonsoft.Json.dll</HintPath>
+    </Reference>
+    <Reference Include="Simplic.CoreLib, Version=6.14.418.1002, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\Simplic.CoreLib.6.14.418.1002\lib\net451\Simplic.CoreLib.dll</HintPath>
+    </Reference>
+    <Reference Include="Simplic.Flow, Version=6.1.1118.1004, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\Simplic.Flow.6.1.1118.1004\lib\net451\Simplic.Flow.dll</HintPath>
+    </Reference>
+    <Reference Include="Simplic.Flow.Event, Version=6.1.118.914, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\packages\Simplic.Flow.Event.6.1.118.914\lib\net451\Simplic.Flow.Event.dll</HintPath>
+    </Reference>
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Xml.Linq" />
+    <Reference Include="System.Data.DataSetExtensions" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Net.Http" />
+    <Reference Include="System.Xml" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="ActionNode\Base\ConsoleWriteLineNode.cs" />
+    <Compile Include="ActionNode\Base\ForeachNode.cs" />
+    <Compile Include="ActionNode\Base\GetVariableNode.cs" />
+    <Compile Include="ActionNode\Math\AddNode.cs" />
+    <Compile Include="ActionNode\Math\DivideNode.cs" />
+    <Compile Include="ActionNode\Math\MultiplyNode.cs" />
+    <Compile Include="ActionNode\Base\SetVariableNode.cs" />
+    <Compile Include="ActionNode\IO\DeleteFileNode.cs" />
+    <Compile Include="ActionNode\IO\GetDirectoryContentNode.cs" />
+    <Compile Include="ActionNode\Math\SubtractNode.cs" />
+    <Compile Include="ActionNode\Process\ProcessStartNode.cs" />
+    <Compile Include="ActionNode\Thread\ThreadSleepNode.cs" />
+    <Compile Include="EventNode\IO\OnCheckDirectoryContentEventArgs.cs" />
+    <Compile Include="EventNode\IO\OnCheckDirectoryContentNode.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="ActionNode\Base\SequenceNode.cs" />
+    <Compile Include="ActionNode\Base\StartWithConditionNode.cs" />
+    <Compile Include="YvzNode.cs" />
```

Comment:
Shouldn't we remove this?

Reply:
yes its somehow included again. removing now